catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -104,17 +106,30 @@ public Filter convertBQSToDDF(String query) {
 
     Filter filter = bqsListener.getFilter();
     if (filter != null && StringUtils.isNotBlank(filter.toString())) {
-      LOGGER.debug("Parsed Query: {}", filter);
+      LOGGER.debug("Parsed Query: {}", prettyPrintFilter(filter));
```

Comment:
These should probably be protected with an if debug level is debug statement - as is, the filter is always transformed into a string regardless of the logging level.

Reply:
They are. You added that check in a later commit 😉